bin/validate-htaccess

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+#
+# This script injects a file into the current Apache2 environment temporarily
+# in order to check its syntax.
+#
+# USAGE:
+#
+#     validate-htaccess <file>
+#
+# EXAMPLE:
+#
+#     validate-htaccess /path/to/some/config/file.conf
+#     validate-htaccess /path/to/.htaccess
+#
+# EXIT CODES:
+#
+# 0 - The syntax is valid.
+# 1 - The syntax is invalid, see STDERR for details.
+# 2 - The httpd binary could not be found.
+# 3 - The given file does not exist.
+# 4 - The current Apache configuration (without including the file being tested)
+#     appears to be invalid, so we cannot safely proceed.
+#
+# Author: Liquid Web
+# License: MIT
+
+file=${1:?No configuration file specified.}
+
+# Ensure that httpd is available.
+if ! command -v httpd &> /dev/null; then
```

Comment:
Not for our use case, but in general - is there an alternative that doesn't require httpd to be installed?

Reply:
Not that I'm aware of, but from what I can tell systems that use `apachectl` are actually using `httpd` under the hood, so `httpd` _should_ be available on anything running Apache2.

Reply:
I'm more specifically thinking about someone who might want to use this tool locally during testing without it having the dependency itself installed.

Reply:
Shy of writing a full linter, I don't think so. It could be handy to wrap this all in a Docker image, however, making it easy for people to test without having to have Apache installed.